### DISCUSSION TITLE
Incorrect repo shown in the generated code

### DISCUSSION THREAD

**Discussion by giscus** [giscus/giscus#943] [General]
# Welcome to giscus!

http://localhost:3000/guestbook

**Comment by nathanbuchar** [comment]
~Hum, the generated code from https://giscus.app/ doesn't seem to be pointing at the correct repo.~

**Reply by laymonage** [reply]
Could you give a screenshot? I don't see a reason why it would point to the giscus repo, unless you wrote `giscus/giscus` in the repository input.

**Reply by nathanbuchar** [reply]
User error. Ignore me, I had a bad copy and paste moment :-)

**Reply by nathanbuchar** [reply]
It works!

https://n8.engineer/guestbook
